### DISCUSSION TITLE
Sonnet Technology McFiver

### DISCUSSION THREAD

**Discussion by PixlRainbow** [geerlingguy/raspberry-pi-pcie-devices#446] [Ideas]
Sonnet Technology just unveiled it's... rather _odd_ combination five-in-one device, the [McFiver](https://www.tomshardware.com/news/sonnet-m2-usb-c-10gb-ethernet-mcfiver-expanison-card), which includes two USB type C ports, two M.2 slots and 10 gigabit Ethernet

Thoughts?

**Comment by geerlingguy** [comment]
It costs $399, which is a bit much to pay for all those devices compared to separate cards... but could be useful in constrained builds (e.g. when you have one slot, expecially in Mini ITX or with a CM4!): https://www.sonnettech.com/product/mcfiver-pcie-card/overview.html

I don't think I'd test one of my own accord, but it would be fun to test if someone could acquire one.
